TestRun/fails_when_the_widget_doesn't_draw seems to be racy

Details:

$ go test -race ./...
--- FAIL: TestRun (0.90s)
    --- FAIL: TestRun/fails_when_the_widget_doesn't_draw (0.01s)
        termdash_test.go:357: Run => unexpected error: <nil>, wantErr: true

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bugfixes in the braille canvas.
 - Lint errors reported on the Go report card.
+- Flaky behavior in termdash_test.
 
 ## [0.4.0] - 15-Jan-2019
 

--- a/termdash_test.go
+++ b/termdash_test.go
@@ -174,7 +174,7 @@ func TestRun(t *testing.T) {
 			},
 		},
 		{
-			desc: "fails when the widget doesn't draw",
+			desc: "fails when the widget doesn't draw due to size too small",
 			size: image.Point{1, 1},
 			opts: []Option{
 				RedrawInterval(1),
@@ -350,7 +350,7 @@ func TestRun(t *testing.T) {
 				t.Fatalf("container.New => unexpected error: %v", err)
 			}
 
-			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Millisecond)
+			ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 			err = Run(ctx, got, cont, tc.opts...)
 			cancel()
 			if (err != nil) != tc.wantErr {

